src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1072,8 +1058,11 @@ define(function(require) {
 			});
 		},
 
-		numbersPaintSpare: function(parent, dataNumbers, callback) {
+		numbersPaintSpare: function(args) {
 			var self = this,
+				parent = args.parent,
```

Comment:
This variable can be removed, as `parent` will be used only once within the function.
```suggestion
```